### DISCUSSION TITLE
thanks a lot for bringing this plugin

### DISCUSSION THREAD

**Discussion by HisoKaaaa44** [JustOptimize/return-ShowHiddenChannels#8] [General]
fr i cant thank you enough for bringing it back to life

**Comment by Heth213** [comment]
thank you!

**Comment by NocturnalKing** [comment]
Agreed, I would also like to say thanks!

**Comment by jeyhaley0-7** [comment]
I don't get anything

**Comment by juliarazalb-design** [comment]
The new version 6.6 has stopped working again.
